code/modules/client/preference/preferences_toggles.dm

```diff
@@ -293,7 +293,7 @@
 /client/verb/toggle_ghost_pda()
 	set name = "Show/Hide GhostPDA"
 	set category = "Preferences"
-	set desc = ".Toggle seeing PDA messages as an observer."
+	set desc = "Toggle seeing PDA messages as an observer."
```

Comment:
Undocumented change.

Reply:
Wasn't sure if I needed to mention that since it was just fixing a typo, but why not.